### DISCUSSION TITLE
How do i disable styling/theme of only the scrollbars ?

### DISCUSSION THREAD

**Discussion by Montago** [MahApps/MahApps.Metro#4365] [Q&A]
My app is mixing the Theme of Telerik Fluent and MahApps Metro ... 

My problem is that Mahapps is overriding the theme of Telerik Fluent - especially in the GridView .

How do i disable only 1 style setting but keep the rest ?

**Comment by punker76** [comment]
Find out which style is used by Telerik and set it explicitly after the MahApps styles in your App.xaml.
